packages/browser/src/providers/bitski-browser-engine.ts

```diff
@@ -69,11 +78,14 @@ export class BitskiBrowserEngine extends BitskiEngine {
       this.addProvider(cacheSubprovider);
     }
 
-    // Respond to requests that need approval with an iframe
-    const iframeSubprovider = new IFrameSubprovider(this.webBaseUrl, this.apiBaseUrl, this.network.chainId, this.tokenProvider, this.headers);
-    this.addProvider(iframeSubprovider);
+    // Ensure that whenever accounts are requested, they go through Bitski
```

Comment:
Note that the cacheSubprovider comes first. Most of the time, eth_accounts should be loading from that.